CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log for coreJSON Library
 
+## v1.1.0 (November 2020)
+
+### Updates
+ - [#53](https://github.com/FreeRTOS/coreJSON/pull/53) Add searching an array by index.
```

Comment:
Probably better to mention the API function of change, and the fact that it is a backwards incompatible change.
```suggestion
   Update `JSON_Search` function to support searching JSON arrays. This change is NOT backwards compatible.
 - [#53](https://github.com/FreeRTOS/coreJSON/pull/53) Add searching an array by index.
 
```

Reply:
Sure